open_rarity/models/token_ranking_features.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class TokenRankingFeatures:
+    """Class represents all standardized ranking features
+    that should be considered by the ranking function.
+
+    Attributes
+    ----------
+    unique_attribute_count : int
+        count of unique attributes in the asset ( 1 of 1)
```

Comment:
nit: remove space after first parenthesis. and 1 of 1 may be confusing - maybe just say
(unique attribute is an attribute name/value pair that only that asset has in the entire collection